sdk/communication/azure-communication-common/README.md

```diff
@@ -20,7 +20,7 @@ This package contains common code for Azure Communication Service libraries.
   APIs that would require the Java 8+ API desugaring provided by Android Gradle plugin 4.0.0.
 
 ### Versions available
-The current version of this library is **1.0.0-beta.4**.
+The current version of this library is **1.0.0-beta.5**.
```

Comment:
We should reference the latest released version in the README in all places, that is `1.0.0-beta.4`, instead of the one currently under development. Maybe we should consider changing the phrasing here to "The latest released version of this library is <version>" or something like that.
```suggestion
The current version of this library is **1.0.0-beta.4**.
```